### PR TITLE
I210 merge

### PR DESCRIPTION
**Pull request by eugenevinitsky**
Merge the I210 network into master so everyone has access to it.

### CODE REVIEW

File: examples/exp_configs/templates/sumo/test.net.xml [hunk 1]
```diff
@@ -0,0 +1,5727 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Yasharzf) [inline examples/exp_configs/templates/sumo/test.net.xml]: should be deleted. also two above xml files should be deleted. @AboudyKreidieh ?

File: examples/exp_configs/templates/sumo/test.taz.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<additional>
```
Comment (Yasharzf) [inline examples/exp_configs/templates/sumo/test.taz.xml]: should be deleted.

File: examples/exp_configs/templates/sumo/test_od.od [hunk 3]
```diff
@@ -0,0 +1,8 @@
+$O;D2
```
Comment (Yasharzf) [inline examples/exp_configs/templates/sumo/test_od.od]: should be deleted.

**Comment by Yasharzf** [comment]
BTW! pydocstyle test has failed!

File: flow/core/experiment.py [hunk 4]
```diff
@@ -1,10 +1,16 @@
...
-import logging
 import datetime
-import numpy as np
+import logging
 import time
 import os
+import sys
+
+if sys.platform == 'darwin':
+    import matplotlib
```
Comment (AboudyKreidieh) [inline flow/core/experiment.py]: can we try the suggestion by @Yasharzf ? i.e. add to the requirments

```
matplotlib==3.1.0
```

**Comment by AboudyKreidieh** [comment]
can you resolve the pep8 and pydocstyle issues?

**Comment by eugenevinitsky** [comment]
Yes for sure, thank you for your other changes.

**Comment by eugenevinitsky** [comment]
I'll add some tests so that this doesn't drop coverage too much and then it looks ready to merge?

**Comment by eugenevinitsky** [comment]
@AboudyKreidieh you down to merge this?

File: examples/train.py [hunk 5]
```diff
@@ -147,7 +147,6 @@ def setup_exps_rllib(flow_params,
     config["lambda"] = 0.97
     config["kl_target"] = 0.02
     config["num_sgd_iter"] = 10
-    config['clip_actions'] = False  # FIXME(ev) temporary ray bug
```
Comment (kanaadp) [inline examples/train.py]: is this fixed when we upgrade to 8.0.0?

File: examples/train.py [hunk 6]
```diff
@@ -182,12 +181,12 @@ def setup_exps_rllib(flow_params,
...
     elif hasattr(module_ma, flags.exp_config):
         submodule = getattr(module_ma, flags.exp_config)
-        assert flags.rl_trainer == "RLlib", \
+        assert flags.rl_trainer.lower() == "RLlib".lower(), \
             "Currently, multiagent experiments are only supported through "\
             "RLlib. Try running this experiment using RLlib: 'python train.py EXP_CONFIG'"
     else:
         assert False, "Unable to find experiment config!"
-    if flags.rl_trainer == "RLlib":
+    if flags.rl_trainer.lower() == "rllib":
```
Comment (kanaadp) [inline examples/train.py]: why rename it? just for aesthetics?

File: flow/core/experiment.py [hunk 7]
```diff
@@ -1,13 +1,12 @@
...
 import datetime
-import numpy as np
+import logging
 import time
 import os
-
-from flow.core.util import emission_to_csv
-from flow.utils.registry import make_create_env
+import matplotlib.pyplot as plt
+import numpy as np
```
Comment (kanaadp) [inline flow/core/experiment.py]: are these ordered correctly?

File: flow/core/experiment.py [hunk 8]
```diff
@@ -54,12 +53,18 @@ class can generate csv files from emission files produced by sumo. These
     ----------
     env : flow.envs.Env
         the environment object the simulator will run
+    custom_callables : [str, lambda]
+        List of strings and lambda functions corresponding to some information we want
+        to extract from the environment. The lambda will be called at each step to extract
+        information from the env and it will be stored in a dict keyed by the str.
```
Comment (kanaadp) [inline flow/core/experiment.py]: maybe add something that they should return a numerical quantity. i think this script breaks if the callable returns None

File: flow/core/kernel/vehicle/traci.py [hunk 9]
```diff
@@ -18,6 +18,11 @@
 WHITE = (255, 255, 255)
 CYAN = (0, 255, 255)
 RED = (255, 0, 0)
+GREEN = (0, 255, 0)
+STEPS = 10
+rdelta = 255 / STEPS
+# smoothly go from red to green as the speed increases
+color_bins = [[int(255 - rdelta * i), int(rdelta * i), 0] for i in range(STEPS + 1)]
```
Comment (kanaadp) [inline flow/core/kernel/vehicle/traci.py]: this is cool, but SUMO does this by default btw :^)

File: tests/fast_tests/test_examples.py [hunk 10]
```diff
@@ -247,6 +253,17 @@ def test_multi_highway(self):
         }
         self.run_exp(multiagent_highway, **kwargs)
 
+    # TODO(@evinitsky) enable
```
Comment (kanaadp) [inline tests/fast_tests/test_examples.py]: Shouldn't leave this in master if it doesn't work

**Comment by AboudyKreidieh** [comment]
still need to write tests but then we can probably merge

**Comment by eugenevinitsky** [comment]
What do you want to test before we merge this?

File: flow/core/kernel/vehicle/traci.py (hunk 9 above)
Comment (eugenevinitsky) [reply flow/core/kernel/vehicle/traci.py]: How do I turn this on in SUMO?

File: tests/fast_tests/test_examples.py (hunk 10 above)
Comment (eugenevinitsky) [reply tests/fast_tests/test_examples.py]: Think it's alright, we have other tests like this that we need to turn on at some point. Not perfect code but this isn't a stable project till a version release anyways

File: flow/core/kernel/vehicle/traci.py (hunk 9 above)
Comment (eugenevinitsky) [reply flow/core/kernel/vehicle/traci.py]: Can you replace this with the correct command?

Comment (kanaadp) [reply flow/core/kernel/vehicle/traci.py]: non-trivial to implement, have to add a gui-settings file and manually set that as the default setting for vehicle coloring

i.e. https://sumo.dlr.de/docs/SUMO-GUI.html#configuration_files

File: examples/train.py (hunk 6 above)
Comment (eugenevinitsky) [reply examples/train.py]: They can just type whatever they want and capitalization errors wont throw an error

File: examples/train.py (hunk 5 above)
Comment (eugenevinitsky) [reply examples/train.py]: yes. 0.7.3 fixes it too

**Comment by eugenevinitsky** [comment]
@AboudyKreidieh I have tests for everything but param_sweep.py

**Pull request merged**